website/docs/getting-started/clean-up.mdx

```diff
@@ -0,0 +1,48 @@
+---
+sidebar_position: 4
+---
+
+import { Screenshot } from "../../src/components/screenshot";
+
+# Clean Up
+
+We can teardown the stack we just created by running:
+
+```bash
+npx cdk destroy
+```
+
+CDK will prompt you to confirm that you'd like to destroy your stack:
+
+import confirmDestroy from "./assets/confirm-destroy.png";
+
+<Screenshot src={confirmDestroy} />
+
+Typing `y <enter>` will confirm that you want to destroy your stack.
+You will see the status update as the stack resources are deleted.
+Once everything has been deleted, you should get a confirmation that your stack has been destroyed:
+
+import stackDestroyed from "./assets/stack-destroyed.png";
+
+<Screenshot src={stackDestroyed} />
+
+:::tip
+
+Cleaning up development stacks is good practice to avoid running into resource limits and to ensure your stack is deployable from scratch.
+
+:::
+
+## Wrapping Up
+
+We've only scratched the surface of what you can build with Functionless and we're hard at work adding new functionality on a weekly basis.
+
+You can read more about the [Step Function][step-function] and [Function][function] Constructs we used in this guide in our concepts documentation.
+This is also a good place to learn about other Constructs and features that were not included in this guide.
+
+If you'd like to keep up with the project, please [star us on GitHub][github].
+We're also [active on discord][discord] if you'd like some help or just want to share something exciting you've built with Functionless.
+
+[github]: https://github.com/functionless/functionless/stargazers
+[discord]: https://discord.com/invite/VRqHbjrbfC
+[step-function]: http://localhost:3000/docs/concepts/step-function/
+[function]: http://localhost:3000/docs/concepts/function/
```

Comment:
don't like inline links?

Reply:
Also, you can use relative urls for content in the docs, by the way (../concepts/function)

Reply:
Doh and these are localhost links. I don't like long links inline as I find it makes it hard to edit/review. Will fix these and use relative links since they are short.

Reply:
I love the links at the bottom. Good reference. Cleans up the content and removes duplicates. Thanks for demonstrating that.